src/QueryDataTable.php

```diff
@@ -156,10 +157,21 @@ public function prepareCountQuery(): QueryBuilder
         $builder = clone $this->query;
 
         if ($this->isComplexQuery($builder)) {
+            $builder->select(DB::raw('1'));
+
+            if (! $this->isComplexQuery($builder)) {
```

Comment:
Can you please review and fix this phpstan findings? Thanks!

Reply:
well I'm not sure how to fix this as PHPStan is wrong

(you can test this by commenting the inside of the if and running the `test_simple_queries_with_complexe_select_are_wrapped_without_selects` test, it will fail)